project-koku/koku
# Atempt to move ocp-gcp filter logic to trino

## Jira Ticket

[COST-3843](https://issues.redhat.com/browse/COST-3843)

## Description

This change will adds a little new logic to move ocp on gcp filter matching logic to trino

## Testing

1. Checkout Branch
2. Restart Koku
3. Add OCP on GCP sources
4. Validate everything runs correctly and cost is what we expect

Alt  you can run QE test test_api_ocp_on_gcp_source_raw_calc

...
